incubator/hnc/internal/kubectl/describe.go

```diff
@@ -67,6 +73,22 @@ var describeCmd = &cobra.Command{
 			fmt.Printf("  No children\n")
 		}
 
+		w := tabwriter.NewWriter(os.Stdout, 0, 0, 1, '  ', 0)
+		fmt.Fprintln(w, "  EVENTS:\n  LAST SEEN\tREASON\tOBJECT\tMESSAGE")
```

Comment:
What does this look like if there aren't any events? Also we don't use all-caps anywhere else in this plugin, does it look ok with regular capitalization?

Reply:
If there's no event, it's just an empty title line - actually I think it might be better just showing ` Events: None`, wdyt?

Will change to regular capitalization

Reply:
Yeah, the empty title line on its own isn't very friendly. `Events: None` could work; `No recent events` might be even friendlier.

Reply:
done